noxfile.py

```diff
@@ -26,7 +26,8 @@ def test(session, django: str):
         f'django~={django}.0',
         'pytest', 'pytest-django',
         'ipdb', 'freezegun',
-        'psycopg2-binary',
+        'psycopg2-binary', 'pyjwt',
+        'cryptography', 'pyopenssl',
         '.',
```

Comment:
Can we just do `session.install('.[apple-in-app]')`? This way you won't need to duplicate dependencies

Reply:
Fair point!

Reply:
Done.